charts/islandis/values.dev.yaml

```diff
@@ -1640,6 +1640,7 @@ search-indexer-service:
       cpu: '100m'
       memory: '512Mi'
   secrets:
+    API_CMS_DELETION_TOKEN: '/k8s/search-indexer/API_CMS_DELETION_TOKEN'
```

Comment:
has this param been added to SSM in all envs?

Reply:
Yep